### PR TITLE
adds a toy ritual dagger

### PR DESCRIPTION
**Pull request by boodaliboo**
### Intent of your Pull Request
adds the new toy ritual dagger to the clerk and clown vendor as contraband a fun new item to prank security with
#### Changelog

:cl:  
rscadd: Added a new toy to prank security with 
/:cl:

### CODE REVIEW

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -306,7 +306,17 @@
 	attack_verb = list("pricked", "absorbed", "gored")
 	w_class = WEIGHT_CLASS_SMALL
 	resistance_flags = FLAMMABLE
-
+	
+/obj/item/toy/toyritualdagger
+	name = "Ritual Dagger"
+	desc = "It looks like one of those toys nar sie cultist give to there children."
```
Comment (patpol4) [inline code/game/objects/items/toys.dm:312]: dude fix the grammar

**Comment by boodaliboo** [comment]
Don’t bully me

**Comment by LoliconSlayer** [comment]
You do know that toy daggers already exist right

File: code/game/objects/items/toys.dm [hunk 2]
```diff
@@ -306,7 +306,17 @@
...
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "render"
+	item_state = "cultdagger
```
Comment (Ktlwjec1) [inline code/game/objects/items/toys.dm:315]: item_state = "cultdagger**"**

**Comment by ynot01** [comment]
this code + toy dagger already exists

**Comment by boodaliboo** [comment]
I don’t like that dagger though that’s why I coded it

**Comment by fluffe9911** [comment]
This is just redundant if you want any toy dagger changes edit the one that already exists you meme

**Pull request closed without merging**